instrumentation/jmx-metrics/javaagent/README.md

```diff
@@ -302,7 +302,7 @@ rules:
         unit: By
         metricAttribute:
           jvm.memory.pool.name	: param(name)
-          jvm.memory.type: lowercase(beanattr(type))
+          jvm.memory.type: lowercase(beanattr(Type))
```

Comment:
[for reviewer] fixing typo in example, without this the `jvm.memory.type` attribute is silently omitted.